language/German/strings.po

```diff
@@ -1390,7 +1396,7 @@ msgstr "Favoriten"
 #: src/frontend/mame/ui/utils.cpp:113
 msgctxt "software-filter"
 msgid "Parents"
-msgstr "Eltern"
+msgstr "Übergeordnete Systeme"
```

Comment:
This one is software list items, not systems.  Also, please check that this message doesn’t overflow the left pane.  There’s limited space.

Reply:
Urgh, OK. Then I suppose sticking with the _Eltern_ terminology is the better solution.

Reply:
Can you just use “Übergeordnete” or some conjugation of it?  Would that fit?  System/software can be inferred from the heading at the top of the screen.

Reply:
Hmm, That's actually a good idea. Thanks !